lib/helpers/redis-stubs.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+var sinon = require('sinon');
+var redis = require('redis');
+var clientStub = {
+  'get': sinon.stub().callsArg(1),
+  'expire': sinon.stub().callsArg(2),
+  'del': sinon.stub().callsArg(1),
+  'set': sinon.stub().callsArg(2)
+};
+
+var redisCreateClientStub = sinon.stub(redis, 'createClient', function(){
+  return clientStub;
+});
+
+exports.redisCreateClientStub = redisCreateClientStub;
+exports.clientStub = clientStub;
```

Comment:
If we were to use this file it shouldn't be in this location. Sinon is a dev dependency and this location is for production code. It should be in the test folder.